resources/dashboards/File-Reconstruction-Dashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+  "title": "File Reconstruction Dashboard",
+  "hits": 0,
+  "description": "",
+  "panelsJSON": "[{\"id\":\"Top-10-Attach-Types-(bar-graph)\",\"type\":\"visualization\",\"size_x\":6,\"size_y\":4,\"col\":1,\"row\":1},{\"id\":\"Top-10-Attachment-Names\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":7,\"row\":1},{\"id\":\"Top-10-Attachment-Types-by-Total-Bytes-Delta\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":7,\"row\":3},{\"id\":\"Top-10-Email-Receivers-by-Attachment-Type\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":10,\"row\":3},{\"id\":\"Top-10-Senders-by-Attachment-Type\",\"type\":\"visualization\",\"size_x\":3,\"size_y\":2,\"col\":10,\"row\":1},{\"id\":\"Attachment-Search\",\"type\":\"search\",\"size_x\":12,\"size_y\":4,\"col\":1,\"row\":5,\"columns\":[\"SenderEmail\",\"ReceiverEmail\",\"AttachFilenameCdispo\",\"AttachSize\",\"AttachType\",\"Application\"],\"sort\":[\"TimeUpdated\",\"desc\"]}]",
```

Comment:
You have to update the dashboard (to use the new filenames) either by recreating and exporting it through the UI, or change the panel ID's here to reflect their new file names